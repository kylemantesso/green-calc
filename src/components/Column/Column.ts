import styled from "styled-components/native";

export const Column = styled.View`
  flex-direction: column;
  justify-content: flex-end;
  flex: 1;
`;