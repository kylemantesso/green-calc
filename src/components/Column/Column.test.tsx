import React from "react";

import renderer from "react-test-renderer";
import { Column } from "./Column";

test("renders correctly", () => {
  const tree = renderer.create(<Column />).toJSON();
  expect(tree).toMatchSnapshot();
});
