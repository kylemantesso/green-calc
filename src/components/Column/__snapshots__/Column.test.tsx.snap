// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexDirection": "column",
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "flex-end",
      },
    ]
  }
/>
`;
